src/models/fhir/maskNricFin.test.ts

```diff
@@ -0,0 +1,25 @@
+import { getData } from "@govtechsg/open-attestation";
+import { HealthCertDocument, PDTHealthCertV2 } from "src/types";
+import { getNricObjV1, getNricObjV2, maskNRIC, NricIdentifier } from ".";
+import PDTV1 from "../../../test/fixtures/v1/example_healthcert_with_nric_wrapped.json";
+import PDTV2 from "../../../test/fixtures/v2/pdt_pcr_with_nric_wrapped.json";
+
+describe("test maskNricFinInPlace function", () => {
+  it("nric should be masked for V1 Healthcert", () => {
+    const data = getData(PDTV1 as any) as HealthCertDocument;
+    const nricIdentifier = getNricObjV1(data) as NricIdentifier;
+
+    expect(nricIdentifier).not.toBeNull();
+    nricIdentifier.value = maskNRIC(nricIdentifier.value);
+    expect(nricIdentifier.value.includes("****")).toBeTruthy();
```

Comment:
Should we make sure we expect the value to be exactly: `S****989Z`?
Just in case the mask function ever does: `S****9098989Z`